src/app/components/request/request.component.ts

```diff
@@ -13,9 +15,22 @@ export class RequestComponent implements OnInit {
   @Output() requestOpened: EventEmitter<RequestId> = new EventEmitter<
     RequestId
   >();
+  requestStatusColor: UserThemeColorPrimary;
+  requestStatusSecondaryColor: UserThemeColorSecondary;
   constructor() {}
 
-  ngOnInit() {}
+
+  ngOnInit() {
+    console.log(this.requestData);
+
+    // TODO: Find out about the other statuses - C & F
+    this.requestStatusColor = 'primaryAF';
+    this.requestStatusSecondaryColor = 'secondaryAF';
+    if (this.requestData.status === RequestStatus.transit) {
+      this.requestStatusColor = 'primaryHL';
+      this.requestStatusSecondaryColor = 'secondaryHL';
```

Comment:
Aren't the colours based on the userType?

Reply:
@sebinbenjamin No. These colors are based on request status.